Policies.md

```diff
@@ -0,0 +1,20 @@
+# WHATWG Policies
+
+## Principles
+
+The WHATWG is guided by [Principles](./Principles.md) of openness, utility, efficiency and freedom.
+
+## Policies
+
+WHATWG participants operate under a number of policies:
+
+* [Intellectual Property Rights Policy](./IPR%20Policy.md)
+* [Workstream Policy](./Workstream%20Policy.md)
+* [Code of Conduct](https://whatwg.org/code-of-conduct)
+* [Working Mode](https://whatwg.org/working-mode)
+
+Additionally, the WHATWG Steering Group was created by the [Steering Group Agreement](./SG%20Agreement.md) and operates under the [Steering Group Policy](./SG%20Policy.md).
+
+<hr>
```

Comment:
At some point we should fix this to match https://github.com/whatwg/whatwg.org/blob/dbcc63269a3230899384168c5047d42440f8d3d9/whatwg.org/irc#L61-L63 , either in the source files or the translation script.

Reply:
Yeah, #78 tracks that.